src/App/Controls/ExtendedCollectionView.cs

```diff
@@ -1,9 +1,24 @@
-﻿using Xamarin.Forms;
+﻿using System.Linq;
+using Xamarin.CommunityToolkit.Converters;
+using Xamarin.Forms;
 
 namespace Bit.App.Controls
 {
     public class ExtendedCollectionView : CollectionView
     {
         public string ExtraDataForLogging { get; set; }
     }
+
+    public class SelectionChangedEventArgsConverter : BaseNullableConverterOneWay<SelectionChangedEventArgs, object>
+    {
+        public override object? ConvertFrom(SelectionChangedEventArgs? value)
+        {
+            if (value == null)
+            {
+                return null;
+            }
+            return value!.CurrentSelection.FirstOrDefault();
+        }
+    }
```

Comment:
why can't you use [ItemSelectedEventArgsConverter](https://learn.microsoft.com/en-us/xamarin/community-toolkit/converters/itemselectedeventargsconverter)?

Reply:
Because the `ExtendedCollectionView` doesn't have an `ItemSelected` event, only `ItemChanged` and if we use the toolkit `ItemSelectedEventArgsConverter` it throws an exception because it can convert the EventArgs object passed.

Reply:
<img width="519" alt="image" src="https://user-images.githubusercontent.com/4648522/211806924-7e9b974e-26dd-48f4-9599-6b1a2c041214.png">

Reply:
oh makes sense just change it to:
```c#
return value?.CurrentSelection.FirstOrDefault();
```
so you avoid checking and returning null before